### PR TITLE
ansible: improve daemon restart

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: curve-ansible/roles/restart_service/tasks/include/restart_nebd.yml [hunk 1]
```diff
@@ -15,9 +15,23 @@
 #  limitations under the License.
 #
 
-- name: restart nebd-server
+- name: set sudo
+  set_fact:
+    sudo: "sudo"
+  when: command_need_sudo
+
+- name: restart nebd-server with daemon
```
Comment (ilixiaocui) [inline curve-ansible/roles/restart_service/tasks/include/restart_nebd.yml:23]: Why not reuse the above restart module

File: curve-ansible/common_tasks/restart_service.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - name: restart by daemon
+      shell: chdir={{ ch_dir }} {{ sudo }} {{ daemon_script }} restart
+      register: outres
+    - debug:
+        msg: "{{ outres.stdout }}"
+  when: start_by_daemon
+
+- name: restart service without daemon
+  block:
+    - name: stop service without daemon
```
Comment (cw123) [inline curve-ansible/common_tasks/restart_service.yml:39]: 这里需要wait until server is down吗？

**Review by cw123 — CHANGES_REQUESTED** [review]

File: curve-ansible/roles/restart_service/tasks/include/restart_nebd.yml (hunk 1 above)
Comment (SeanHai) [reply curve-ansible/roles/restart_service/tasks/include/restart_nebd.yml:23]: > Why not reuse the above restart module

done

File: curve-ansible/common_tasks/restart_service.yml (hunk 2 above)
Comment (SeanHai) [reply curve-ansible/common_tasks/restart_service.yml:39]: > Is here need wait until server is down？

done

**Review by ilixiaocui — APPROVED** [review]

**Review by cw123 — APPROVED** [review]

**Pull request merged**